src/components/DialogCard.vue

```diff
@@ -0,0 +1,109 @@
+<script lang="ts">
+import { defineComponent, computed } from 'vue';
+
+// Component used to render dialog window which you can fill with custom content
+// Available slots:
+// - title
+// - metadata
+// - content
+// - buttons
+// - image
+// The content and image are rendered side by side in a scrollable dialog window
+export default defineComponent({
+  name: 'DialogCard',
+  props: {
+    modelValue: {
+      type: Boolean,
+      required: true,
+    },
+  },
+  emits: ['update:modelValue', 'change'],
+  setup(props, { emit }) {
+    const dialogOpen = computed({
+      get: () => props.modelValue,
+      set: (value) => {
```

Comment:
Missing func input `value` param type and return value type `set: (value: boolean): void => {`

Reply:
Added.